.travis.yml

```diff
@@ -0,0 +1,37 @@
+# Sets the build environment.
+sudo: required
+dist: trusty
+
+language: cpp
+
+addons:
+  apt:
+    packages:
+      - libboost1.54-all-dev
+      - libxi-dev
+
+matrix:
+  include:
+  #  - env: PYTHON="3.5" CONDA_PY=35 ARCH="x86_64"
+  #    os: linux
+  #    dist: trusty
+    - env: PYTHON="3.6" CONDA_PY=36 ARCH="x86_64"
+      os: linux
+      dist: trusty
+
+before_install:
+  - git submodule update --init --remote --recursive
```

Comment:
remove this line...can add "git clone https://github.com/pybind/pybind11.git pybind11" if we need to grab the latest master..but i don't have this in appveyor and it's causing the builds to crash